src/components/tile/tile.stories.mdx

```diff
@@ -0,0 +1,273 @@
+import { Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Tile from '.';
+import Content from '../content';
+import { info, notes } from './documentation';
+import getDocGenInfo from '../../utils/helpers/docgen-info';
+import { dlsThemeSelector } from '../../../.storybook/theme-selectors';
+import { action } from '@storybook/addon-actions';
+import Dl from '../definition-list/dl.component';
+import Dt from '../definition-list/dt.component';
+import Dd from '../definition-list/dd.component';
+import Link from '../link/link.component';
+import Button from '../button/button.component';
+import Heading from '../heading/heading';
+
+<Meta title="Design System/Tile" parameters={{ info: { disable: true }}} />
+
+# Tile
+Tiles contain static content for visual scanning only, and don't contain controls.
+
```

Comment:
Sorry to be a pain, and I know I haven't done this on Checkbox (I should have), could you add a line here saying that the following spacing props are available: 
![image](https://user-images.githubusercontent.com/14963680/92239572-2a08d500-eeb3-11ea-8b80-cac8c50bdc76.png) and that they can either be a number (multiple of 8px) or any valid CSS string please?